itou/templates/apply/process_accept.html

```diff
@@ -38,29 +38,17 @@ <h3 class="h2">Embauche</h3>
 
         </div>
 
+        <hr class="my-5">
         {% buttons %}
-            {% if job_application.to_siae.is_subject_to_eligibility_rules %}
-                <h4 class="h3 text-center mt-5">Obtention d'un PASS IAE</h4>
-                <p class="text-center pt-2">Dans le cadre de cette embauche, souhaitez-vous obtenir un PASS IAE ?</p>
-                <div class="form-group">
-                    <button type="button" class="btn btn-success btn-block" data-toggle="modal" data-target="#confirmModal">
-                        Oui, j'ai besoin d'une aide au poste
+            <div class="row align-items-center mb-4">
+                <div class="col"></div>
+                <div class="col-auto">
+                    <a class="btn btn-outline-primary bg-white" href="{% url 'apply:details_for_siae' job_application_id=job_application.id %}">Retour</a>
```

Comment:
Je crois que tu cherches `btn-link` :).
```suggestion
                    <a class="btn btn-link" href="{% url 'apply:details_for_siae' job_application_id=job_application.id %}">Retour</a>
```

Reply:
Bon, j'ai testé ça ne marche pas, il faut bien laisser btn-outline-primary bg-white